plugins/woocommerce/tests/php/src/Internal/FraudProtection/CheckoutEventTrackerTest.php

```
@@ -154,43 +125,38 @@ function ( $event_data ) {
 			->method( 'get_current_shipping_country' )
 			->willReturn( null );
 
-		// Mock scheduler to capture event data.
 		$captured_event_data = null;
-		$this->mock_dispatcher
+		$this->mock_collector
 			->expects( $this->once() )
-			->method( 'dispatch_event' )
+			->method( 'collect' )
 			->willReturnCallback(
 				function ( $event_type, $event_data ) use ( &$captured_event_data ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$event_type'. (undefined)

(UnusedFormalParameter)